sql/core/src/test/scala/org/apache/spark/sql/test/TestSQLContext.scala

```diff
@@ -34,6 +34,9 @@ private[spark] class TestSparkSession(sc: SparkContext) extends SparkSession(sc)
     this(new SparkConf)
   }
 
+  SparkSession.setDefaultSession(this)
```

Comment:
Intentional conflict with https://github.com/apache/spark/pull/20926 , should resolve once one is merged

Reply:
We will merge that one first.